plugins/kubernetes/src/components/DeploymentsAccordions/DeploymentsAccordions.tsx

```diff
@@ -176,13 +171,19 @@ const DeploymentAccordion = ({
         />
       </AccordionSummary>
       <AccordionDetails>
-        <PodsTable pods={ownedPods} extraColumns={deploymentPodColumns} />
+        <PodsTable
+          pods={ownedPods}
+          extraColumns={deploymentPodColumns}
+          podTableColumns={podTableColumns}
```

Comment:
should `extraColumns` and `podTableColumns` be the same thing? Similar to the catalog which just has one prop passed in which you can override completely (recreating the default columns as you find necessary)

either that or should we merge the two objects and just continue to pass in `extraColumns` as a single prop?

Reply:
See convo below, merged them and passing them as a single prop now